be/src/storage/lake/filenames.h

```diff
@@ -63,8 +63,8 @@ inline std::string tablet_metadata_filename(int64_t tablet_id, int64_t version)
     return fmt::format("{:016X}_{:016X}.meta", tablet_id, version);
 }
 
-inline std::string tablet_delvec_filename(int64_t tablet_id, int64_t version) {
-    return fmt::format("{:016X}_{:016X}.delvec", tablet_id, version);
+inline std::string tablet_delvec_filename() {
```

Comment:
if is a random name, may want to rename it to `random_tablet_delvec_filename()` or `generate_delvec_filename()`?